modules/nupac.nu

```diff
@@ -11,11 +11,6 @@
 #?keywords: [package, management]
 #?os: [android, linux, macos, windows]
 
```

Comment:
Please bring it back. I thought we agreed on using verbose, auto-generated `nupac version`, while using the hardcoded one in `export def nupac`

```suggestion

# nupac's version
def VERSION [] {
    "0.1.0"
}
```

Reply:
There is really no value in displaying a hardcoded version that's an exact duplicate of the same version we store few lines above

Reply:
If you insist, metadata will be moved to a separate file tho